tests/nvmeof/workflows/ha.py

```diff
@@ -446,6 +460,127 @@ def power_on_off(self, gateway, action, wait_for_active_state=None):
 
         return False
 
+    def scale_down(self, gateway_nodes):
+        """Scaling down of the NVMeoF Gateways.
+
+        Initiate scale-down
+        - List the gateways which has to be scaled down.
+        - Validate the ANA states of scaled down GWs are optimized in one of the other working GWs.
+
+        Post scale-down Validation
+        - List out namespaces associated with the scaled down Gateways using ANA group ids.
+        - Check for 5 Consecutive times for the increments in write/read to validate IO continuation.
+        """
+        fail_gws, _ = self.catogorize(gateway_nodes)
+        start_counter = float()
+        start_time = str()
+        end_counter = float()
+        end_time = str()
```

Comment:
Seems like counters are not getting updated? 

2024-10-11 19:43:02,753 (cephci.test_ceph_nvmeof_loadbalancing) [INFO] - cephci.cephci.tests.nvmeof.workflows.ha.py:574 - [ ceph-nvme-8-2-mn0w9x-node10 ] Total time taken to scale up - 1280664.256041291 seconds
2024-10-11 19:43:02,753 (cephci.test_ceph_nvmeof_loadbalancing) [INFO] - cephci.cephci.tests.nvmeof.workflows.ha.py:856 - {
    "scale-up-start-time": "",
    "scale-up-end-time": "Fri Oct 11 19:43:02 2024",
    "scale-up-start-counter-time": 0.0,
    "scale-up-end-counter-time": 1280664.256041291
}

Reply:
Done